atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/WrappingQueryRunner.java

```diff
@@ -54,8 +54,8 @@ Void batchMutate(
                 return null;
             });
         } catch (UnavailableException e) {
-            throw new InsufficientConsistencyException(
-                    "This batch mutate operation requires {} Cassandra nodes to be up and available.",
+            throw CassandraTExceptions.mapToUncheckedException(
+                    "Batch mutate operation requires Cassandra nodes to be up and available. Check these nodes: ",
```

Comment:
here and below: It's probably confusingly named, but the argument we have is a consistency level, which says how many nodes are required to be up (usually this is either QUORUM or ALL). Maybe we should say "Check that sufficiently many nodes are up." or something like that?

Reply:
How about "x operation requires a sufficient number of nodes to be up. Check that the consistency level of {QUORUM or ALL} is adequate."?

Reply:
Made the change to the above in the latest commit let me know if it doens't work